### PR TITLE
Upcoming meetups to Meetuplocation list page

### PR DESCRIPTION
**Pull request by sa7i**
#264 
This PR adds upcoming meetups list to the meetup location list page.
To test this, add a few upcoming meetups from the admin panel, or add it normally.
image

EDIT:
image

### CODE REVIEW

File: systers_portal/meetup/tests/test_views.py [hunk 1]
```diff
@@ -45,25 +45,38 @@ def test_view_meetup_location_about_view(self):
...
+        """Test Meetup Location list view for correct http response,
+        all meetup locations in a list and all upcoming meetups"""
         url = reverse('list_meetup_location')
         response = self.client.get(url)
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, "meetup/list_location.html")
         self.assertContains(response, "Foo Systers")
+        self.assertContains(response, "Foo Bar Baz")
         self.assertContains(response, "google.maps.Map")
+        self.assertEqual(len(response.context['object_list']), 1)
```
Comment (tapasweni-pathak) [inline systers_portal/meetup/tests/test_views.py]: Add comments on which tests are testing the three that you described above.

File: systers_portal/meetup/views.py [hunk 2]
```diff
@@ -31,6 +31,12 @@ class MeetupLocationList(ListView):
     model = MeetupLocation
     paginate_by = 20
 
+    def get_context_data(self, **kwargs):
+        context = super(MeetupLocationList, self).get_context_data(**kwargs)
+        context['meetup_list'] = Meetup.objects.filter(
+            date__gte=datetime.date.today()).order_by('date', 'time')
```
Comment (tapasweni-pathak) [inline systers_portal/meetup/views.py]: Django `datetime.date.today()` cause issues w/ timezones, did you test w/ setting a different timezone in your machine?

File: systers_portal/static/css/style.css [hunk 3]
```diff
@@ -354,6 +354,19 @@ table.decoration-none  tr a {
...
+.text-bottom{
+    padding: 10px;
+    border-bottom: 1px solid #eee;
+}
+
+.upcomingMeetupslist{
+  height: 400px;
+  overflow-y: scroll;
+  line-height: 1em;
+  word-wrap: break-word;
```
Comment (tapasweni-pathak) [inline systers_portal/static/css/style.css]: What do you see if you use `overflow-wrap` in place of `word-wrap`?

Comment (sa7i) [reply systers_portal/static/css/style.css]: Both results in the same, like how it looks in the screenshot. "overflow-wrap: word-wrap property has been renamed overflow-wrap in the current draft of the CSS3 Text specification" Should I modify it to overflow-wrap?

**Comment by MeepyMay** [comment]
@sa7i @tapasweni-pathak @mansimarkaur What's the status with this PR?

**Pull request merged**